addons/binding/org.openhab.binding.edimax/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="edimax"
```

Comment:
Edimax has many other products like cameras etc.
Would "edimaxsmartplug" maybe be the better binding name?